### DISCUSSION TITLE
Is it possible to use TipTap if vue is coming in through a CDN?

### DISCUSSION THREAD

**Discussion by plannedWin** [ueberdosis/tiptap#4054] [General Questions]
I am currently bringing vue in through a CDN and it's not working out. 

I keep getting the error:
"Missing ref owner context. ref cannot be used on hoisted vnodes. A vnode with ref must be created inside the render function."

Wondering if there is a way to solve this issue as I love this editor.

Vue version: vue@3

Thanks for the help in advanced.

**Comment by olegvakulenko** [comment]
I'm currently dealing with a similar problem and I'm getting this exact error.

I have a rough idea of ​​what the problem is and I think I have a working solution. It requires some modification to the appropriate editor module.

Not sure if you still need this. I still need to tweak my version a bit.
